frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/advanced-section.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Checkbox,
+  CheckboxProps,
+  ExpandableSection,
+  ExpandableSectionProps,
+  FormGroup,
+} from '@patternfly/react-core';
+import { DeploymentType } from '../../../../constants/create-storage-system';
+import { WizardDispatch, WizardState } from '../../reducer';
+import './backing-storage.scss';
+
+export const AdvancedSection: React.FC<AdvancedSelectionProps> = ({
+  deployment,
+  isAdvancedOpen,
+  dispatch,
+}) => {
+  const { t } = useTranslation();
+
+  const handleSelection: CheckboxProps['onChange'] = (checked) => {
+    dispatch({
+      type: 'backingStorage/setDeployment',
+      payload: checked ? DeploymentType.MCG : DeploymentType.ALL,
+    });
+  };
+  const handleExpanadableToggling: ExpandableSectionProps['onToggle'] = (isExpanded: boolean) => {
+    dispatch({ type: 'backingStorage/setIsAdvancedOpen', payload: isExpanded });
```

Comment:
Not really. This is PF expandable section, the handler passes `isExpanded` the state of expand - true/false .
If its expanded it will be true or false. We just leveraging this to set state of that in our wizard.
if its opened then it should be opened, even if the user navigates back and next.

Reply:
ack